index.html

```diff
@@ -46,113 +46,32 @@ <h2>
         My Recent Works
       </h2>
       <hr>
-      <div class="work-grid">
-        <article class="work-card">
-          <div class="work-card-img">
-          </div>
-          <div class="work-card-content">
-            <h3> 
-              Multi-Post Stories Gain+Glory 
-            </h3>
-            <ul class="tags">
```

Comment:
- [ ] Please make  sure that your design matches Figma design 

| Your design | Figma design |
| ---------------- | ------------------ |
| ![image](https://user-images.githubusercontent.com/74744787/180577422-3bc9cacd-898d-451a-a3da-0077b89693c9.png) | ![image](https://user-images.githubusercontent.com/74744787/180577466-1dc4bb99-5408-4e75-95a9-98058946a421.png) |